lib/swoosh/adapters/gmail.ex

```diff
@@ -99,7 +99,10 @@ defmodule Swoosh.Adapters.Gmail do
     |> prepare_attachments(email)
     |> prepare_reply_to(email)
     |> Mail.Renderers.RFC2822.render()
-    |> parse_bcc(email)
+    # When message is rendered, bcc header will be removed and we need to prepend bcc list to the
+    # begining of the message. Gmail will handle it from there.
+    # https://github.com/DockYard/elixir-mail/blob/master/lib/mail/renderers/rfc_2822.ex#L161
```

Comment:
```suggestion
    # https://github.com/DockYard/elixir-mail/blob/v0.2.0/lib/mail/renderers/rfc_2822.ex#L139
```

Master changes, link to master is not reliable. Also, too bad they haven't made a release since 2017, although they are doing things on the lib.